engine/src/main/java/org/terasology/rendering/ShaderManagerLwjgl.java

```diff
@@ -106,7 +105,7 @@ public void initShaders() {
         // TODO: Find a better way to do this
         prepareAndStoreShaderProgramInstance("post", new ShaderParametersPost());
         prepareAndStoreShaderProgramInstance("ssao", new ShaderParametersSSAO());
-        prepareAndStoreShaderProgramInstance("lightShafts", new ShaderParametersLightShafts());
```

Comment:
I probably worked on two Nodes together, then commited half the files at once and the other half in the next commit to make things easier to review. But yeah, noted for future :)